woocommerce/woocommerce-android
# [HACK] A new CI step to check if dependencies were changed

### Description

The PR adds a new CI step that checks if any dependencies (including transitive) were changed and makes a comment on the PR with a list of the dependencies

*More details will be published tomorrow in a P2*

### Testing instructions

* Try to add/remove/change the version of any dependency in the project
* Push the changes
* Wait for CI and notice a comment in this PR
* Revert your change
* Notice that comment is removed
